influxdb/tests/integration_tests.rs

```diff
@@ -140,7 +140,9 @@ async fn test_wrong_authed_write_and_read() {
             let write_result = client.query(write_query).await;
             assert_result_err(&write_result);
             match write_result {
-                Err(Error::AuthorizationError) => {}
+                Err(Error::ApiError(code)) => {
+                    dbg!(code);
```

Comment:
Debugging the code here is not helpful. We want a test case that fails in CI if it is incorrect, so you'll need to match for the specific error code we expect, and panic otherwise.

Reply:
Did that for own "debugging", and mistakenly commited it...